src/frameworks/drupal7/drush.md

```diff
@@ -1,82 +1,22 @@
-# Work with Drush
+# Working with Drush in Drupal 7
 
-Drush is a command-line shell and scripting interface for Drupal, a
-veritable Swiss Army knife designed to make life easier for those who
-spend their working hours hacking away at the command prompt. Drush
-commands can, for example, be used to clear the Drupal cache, run module
-and database updates, revert features, perform database imports and
-dumps, and a whole lot more. You can reference the full set of Drush
-commands at [Drush.org](http://www.drush.org). If you have never used
-Drush before, you can learn more about it on the [Drush GitHub
-Repository](https://github.com/drush-ops/drush#description)
+Drush is a command-line shell and scripting interface for Drupal, a veritable Swiss Army knife designed to make life easier for those who spend their working hours hacking away at the command prompt. 
 
-You can use the CLI to set up Drush aliases, to easily run Drush commands on
-specific remote Platform.sh environments.
+You can use the CLI to set up Drush aliases, to easily run Drush commands on specific remote Platform.sh environments.
 
-> **note**
-> Platform's CLI requires **Drush 6 or greater**.
+See the [documentation on Drush in Drupal 8](frameworks/drupal8/drush.html) for installation, drush aliases, and other general information.  The installation procedure is the same for both Drupal 7 and 8.
```

Comment:
I think the link needs a `/` prefix, is that right?

Reply:
Yep.  Fixed.